libs/angular/src/lib/toggle-selector/toggle-selector.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-container
+  *ngFor="let item of displayList; let idx = index; trackBy: trackByFn"
+>
+  <div
+    class="ac toggle-selector"
+    [class.error]="!!errorMessage"
+    [class.checked]="item.selected"
+    [class.disabled]="disabled || item.optionItem.disabled"
+    [class.focus]="focusedItem === idx"
+    [class.icon-right]="item.optionItem.iconPosition === 'right'"
+    [class.margin-bottom]="!item.optionItem.description"
+    [class.custom-label]="item.optionItem.customLabel"
+    (click)="handleItemOnClick($event, idx)"
+  >
+    <input
+      [type]="inputType"
+      [attr.aria-label]="item.optionItem.label"
+      [id]="item.optionItem.value"
+      [name]="name"
+      [disabled]="disabled || item.optionItem.disabled"
+      (focus)="focus(idx)"
+      (blur)="blur()"
+      [defaultChecked]="item.selected"
+    />
+    <div
+      class="icon-container"
+      [class.icon-only]="!item.optionItem.label"
+      *ngIf="item.optionItem.icon"
+      [innerHTML]="item.optionItem.icon | safeHtml"
```

Comment:
Is there some way we can avoid using unsanitized html here? Especially considering this value comes from an input that could potentially contain compromised data.
I think we should either limit icon usage to a predefined list, or write a custom sanitizer that only allows whatever html is required to render icons.